partner_credit_limit/model/partner.py

```diff
@@ -58,6 +58,7 @@ def _get_overdue_credit(self):
             # credit = 0.0
             debit_maturity, credit_maturity = 0.0, 0.0
             for line in movelines:
+                limit_day = line.date_maturity
                 if line.date_maturity and line.partner_id.grace_payment_days:
```

Comment:
I changed it, just leave the condition of
If line.partner_id.grace_payment_days